monkey/infection_monkey/exploit/hadoop.py

```diff
@@ -72,9 +72,8 @@ def exploit(self, url, command):
         resp = json.loads(resp.content)
         app_id = resp["application-id"]
         # Create a random name for our application in YARN
-        safe_random = SystemRandom()
         rand_name = ID_STRING + "".join(
-            [safe_random.choice(string.ascii_lowercase) for _ in range(self.RAN_STR_LEN)]
+            [random.choice(string.ascii_lowercase) for _ in range(self.RAN_STR_LEN)]  # noqa: DUO102
```

Comment:
Add a comment explaining why we're ignoring DUO102. Because SystemRandom() blocks on Linux.